### ISSUE TITLE
Date formatting

### ISSUE THREAD

**Issue by SeppNel** [Receipt-Wrangler/receipt-wrangler-api#306]
Currently all dates are showed in Month/Day/Year format. I suggest to  add an option to display it in other formats.

**Comment by Noah231515** [comment]
Absolutely. This is planned for development in December since I'll be away for November.

**Comment by FloatingLizard** [comment]
Still affected by this in June 2026. Running the latest version (6.x) self-hosted on Docker.
For context, I'm using Receipt Wrangler via the API with an AI assistant (Open WebUI), and the MM/DD/YYYY format causes incorrect dates to be stored on every non-US receipt scan. This affects both manual entry and AI-powered quick scan.
Would love to see this make it into an upcoming release. Happy to test if a fix becomes available.
